modules/sno-du-disabling-network-diagnostics.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-disabling-network-diagnostics_{context}"]
+= Disabling network diagnostics
+
+
```

Comment:
@imiller0 This is one of the new topics. Please provide an intro sentence here - basically, why they are using this procedure.

Reply:
added.